images/win/scripts/Installers/Validate-Boost.ps1

```diff
@@ -13,10 +13,10 @@ function Validate-BoostVersion
 
     $ReleasePath = Join-Path -Path $BoostRootPath -ChildPath $BoostRelease
 
-    if ((Test-Path "$ReleasePath\b2.exe") -and (Test-Path "$ReleasePath\bjam.exe"))
```

Comment:
Sorry If I've missed that, but could you please clarify why Boost.Jam was removed?

Reply:
After Boost 1.69.0, bjam is no longer placed in boost root folder. We still copy bjam executable during the artifact generation, just to have consistency with previous versions, but seems like there is no need to check it existence.